eo-maven-plugin/src/main/java/org/eolang/maven/SafeMojo.java

```diff
@@ -343,6 +345,8 @@ private void execWithTimeout() throws ExecutionException, TimeoutException {
                 ),
                 ex
             );
+        } finally {
+            service.shutdown();
```

Comment:
@c71n93 usually, it's done differently: https://stackoverflow.com/questions/10504172/how-to-shutdown-an-executorservice

Reply:
@yegor256 done. WDYT?

Reply:
@yegor256 could you check, please?